api/preprints/views.py

```diff
@@ -142,6 +143,7 @@ class PreprintList(JSONAPIBaseView, generics.ListCreateAPIView, ODMFilterMixin):
         drf_permissions.IsAuthenticatedOrReadOnly,
         base_permissions.TokenHasScope,
         ContributorOrPublic,
+        PreprintPublishedOrAdmin,
```

Comment:
This wasn't being applied on a normal `GET`, required using `ListFilterMixin` and manually checking permissions.